tensorflow/python/keras/utils/losses_utils.py

```diff
@@ -255,7 +255,8 @@ def compute_weighted_loss(losses,
     # to multiple replicas. Used only for estimator + v1 optimizer flow.
     ops.get_default_graph()._last_loss_reduction = reduction  # pylint: disable=protected-access
 
-    if not isinstance(losses, keras_tensor.KerasTensor):
+    if not isinstance(
```

Comment:
This is a good example of a place that prevents Keras from supporting CompositeTensor losses in general right now (regardless of what behavior that loss should have)

Rather than hard-coding `RaggedTensor` here could you try using `composite_tensor.CompositeTensor`? That'll increase the chance that this will generalize to other extension types beyond just RaggedTensors.